recipes/xmlschema/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "xmlschema" %}
+{% set version = "0.9.13" %}
+{% set file_ext = "tar.gz" %}
+{% set sha256 = "5ce11a4b214f7f67962d6a5abff11e7772bc7ecdd70058064c750e623a5c4c53" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ file_ext }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
Consider making `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
Thanks! I just added that line and squashed all the commits.